src/SecretsApi/SecretsApi.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace Pantheon\Terminus\SecretsApi;
+
+class SecretsApi
```

Comment:
I should have thought to suggest in advance that we should examine the PHP output from OpenAPI before mocking this interface, as ultimately, that's what we will be using.

This will do as a placeholder for now, though.